src/epub/css/core.css

```diff
@@ -99,12 +99,6 @@ i > em{
 	font-style: normal;
 }
 
-ol,
```

Comment:
This block was added as part of the new core.css, can you restore it?

Reply:
Oops, didn’t realize that. I’ll revert.